### PR TITLE
Footer: ticket link now linking to ticket area on start page

### LINKED ISSUE #701: Footer menu links not working on all pages
* [x] About page not correctly linked from all pages
* [ ] Ticket link should link to "ticket area" on start page
* [x] Scroll to top should use scrolling feature

screenshot from 2016-08-26 15 15 15

### PR DESCRIPTION
**Pull request by shubham-padia**
intends to resolve #701 .

### CODE REVIEW

File: src/backend/templates/event.hbs [hunk 1]
```diff
@@ -57,6 +57,7 @@
            <div class="col-md-4">
            </div>
            <div style="padding-right:30px" class="col-md-8">
+             <a id="ticket-button" style="display: block ; position: relative ; top: -87px ; visibility: hidden; "></a>  
              <a href="{{{eventurls.register}}}" class="pull-right btn btn-lg btn-primary" role="button">Tickets</a>
```
Comment (championswimmer) [inline src/backend/templates/event.hbs]: could have just given this an id

**Comment by championswimmer** [comment]
@shubham-padia just give the ticket button the id "ticket-button", that will do it. extra anchor tag not needed

**Comment by shubham-padia** [comment]
@championswimmer extra anchor tag is needed to enable the smooth scrolling on the main page

**Comment by shubham-padia** [comment]
@championswimmer And the inline-css in the anchor tag compensates for the header navbar, so that ticket button can be viewd below the navbar when the user clicks on it.

File: src/backend/templates/event.hbs (hunk 1 above)
Comment (shubham-padia) [inline src/backend/templates/event.hbs]: extra anchor tag is needed to enable the smooth scrolling ( like the scrolling that occurs when you click the back to top link ) on the main page.

**Pull request merged**